packages/backend/server/src/base/redis/instances.ts

```diff
@@ -53,3 +53,15 @@ export class SocketIoRedis extends Redis {
     super({ ...config.redis, db: (config.redis.db ?? 0) + 3 });
   }
 }
+
+@Injectable()
+export class QueueRedis extends Redis {
+  constructor(config: Config) {
+    super({
+      ...config.redis,
+      db: (config.redis.db ?? 0) + 4,
```

Comment:
The default maximum value for Redis databases is 16, so a check must be added here to ensure that after adding 4, it does not exceed the maximum value of 15.

https://redis.io/docs/latest/operate/oss_and_stack/management/config-file/

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/hTwOityLamd4hitrae7M/e95b71af-5dad-41c2-a544-115ba1641249.png)